packages/ethers/src/__tests__/mocks/mock-signer.ts

```diff
@@ -0,0 +1,13 @@
+import type { Address, Hex, SmartAccountSigner } from "@alchemy/aa-core";
+
+export class MockSigner implements SmartAccountSigner {
```

Comment:
mocked like how zerodev did, in order to avoid using .env variables like owner nemonic for unit tests so devs can run the unit tests locally. I couldn't use a dummy nemonic like in local-account because the checksum invalid with the ethers library.

Reply:
For unit tests, I think we should use some kind of hard coded mnemonic or private key like https://github.com/wagmi-dev/viem/blob/main/src/_test/constants.ts#L6C8-L6C74 

The problem with hard coding the `signMessage` here is that it won't catch issues with the underlying signer. The sign message tests are actually there because at some point viem wasn't signing `hash` messages correctly so I think we should always use the underlying signer type for that